src/Symfony/Component/Form/Tests/AbstractBootstrap3HorizontalLayoutTest.php

```diff
@@ -100,6 +100,16 @@ public function testLabelWithCustomTextAsOptionAndCustomAttributesPassedDirectly
         );
     }
 
+    public function testLabelNoDivIfFalse()
+    {
+        $this->assertEmpty(
```

Comment:
I honestly don't see the interest. You create a variable that will be used only once.

Why not simply pass the result to the function?